packages/vite-plugin-cloudflare/src/miniflare-options.ts

```diff
@@ -25,6 +26,10 @@ import type {
 import type { MiniflareOptions, SharedOptions, WorkerOptions } from "miniflare";
 import type { FetchFunctionOptions } from "vite/module-runner";
 
+const nodeBuiltInModules = new Set(
+	builtinModules.concat(builtinModules.map((m) => `node:${m}`))
+);
+
```

Comment:
Can we export this from `cloudflare-environment.ts` as it's basically the same (and doesn't necessarily need to be a `Set`)?

Reply:
exporting it from `cloudflare-environment.ts` feels incorrect to me, I can move it to a third file imported by both if that's ok with you?

PS: it's very minimal but I do like it being a set here since it will speed up (very trivially) the lookup below

Reply:
I've moved this to the `utils.ts` file: https://github.com/cloudflare/workers-sdk/pull/8104/commits/3ce2e36db36cd86d339919ef3602f9b17f72ea3b

(and kept is a `Set` 🙈)

I hope this works for you? 🙂